src/components/AssetAccountDetails.tsx

```diff
@@ -30,18 +29,6 @@ export const AssetAccountDetails = ({ assetId, accountId }: AssetDetailsProps) =
   const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
   const assetIds = useMemo(() => [assetId], [assetId])
 
-  const clearAmounts = useSwapperStore(state => state.clearAmounts)
-  const updateBuyAsset = useSwapperStore(state => state.updateBuyAsset)
```

Comment:
As per your comment, lets rip out `updateBuyAsset` and `updateSellAsset` - great catch.